ironic/drivers/modules/drac/job.py

```diff
@@ -72,6 +72,9 @@ def list_unfinished_jobs(node):
     """
     client = drac_common.get_drac_client(node)
 
+    if client.is_lifecycle_in_recovery():
```

Comment:
Why do we need this?

Reply:
We have added this statement in list_unfinished_jobs(node) function because,  when client.list_jobs() get invoked then we are getting "timed out error".
As lifecycle is in recovery mode , we are not able to invoke any WSMan API call to get the jobs list.
That's why we have come up with this workaround that is checking if the lifecycle is in recovery then we are returning from that function.